integration/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "nodejs-docker-integration",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "gulp test"
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@types/mocha": "^2.2.42",
+    "@types/node": "^8.0.26",
+    "@types/request": "^2.0.3",
+    "@types/uuid": "^3.4.1",
+    "gulp": "^3.9.1",
+    "gulp-mocha": "^4.3.1",
+    "gulp-sourcemaps": "^2.6.1",
+    "gulp-typescript": "^3.2.2",
+    "mocha": "^3.5.0",
```

Comment:
`Do you expect to be able to run with `mocha` stand alone or is `gulp-mocha` sufficient?

Reply:
Stand alone `gulp-mocha` is enough and so I removed `mocha` as a dependency.